bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx

```diff
@@ -72,12 +73,17 @@ class ActionsDropdown extends Component {
         </DropdownTrigger>
         <DropdownContent placement="top left">
           <DropdownList>
-            <DropdownListItem
-              icon="presentation"
-              label={intl.formatMessage(intlMessages.presentationLabel)}
-              description={intl.formatMessage(intlMessages.presentationDesc)}
-              onClick={this.handlePresentationClick}
-            />
+            {
+              [
+                (<DropdownListItem
+                  key={_.uniqueId('actions-list-item-')}
```

Comment:
This shouldn't be in the render.
Since we have just 1 item in the Actions List so far - generate a unique key in the componentDidMount and use it here, please.